ui/src/composables/private/__tests__/use-field.cy.js

```diff
@@ -238,112 +418,133 @@ describe('use-field API', () => {
-      it.skip(' ', () => {
-        //
-      })
-    })
+    const slots = [ 'prepend', 'append', 'before', 'after', 'label' ]
 
-    describe('(slot): append', () => {
-      it.skip(' ', () => {
-        //
-      })
-    })
-
-    describe('(slot): before', () => {
-      it.skip(' ', () => {
-        //
-      })
-    })
-
-    describe('(slot): after', () => {
-      it.skip(' ', () => {
-        //
-      })
-    })
+    describe('(slot): default', () => {
+      it('should use the default slot', () => {
+        const model = ref(selectOptions[ 0 ])
+        const labelSlot = 'Hello world'
 
-    describe('(slot): label', () => {
-      it.skip(' ', () => {
-        //
-      })
-    })
+        mountQFieldWrapper({
+          props: {
+            ...vModelAdapter(model),
+            options: selectOptions,
+            labelSlot: true
+          },
+          slots: {
+            default: () => labelSlot
+          }
+        })
 
-    describe('(slot): error', () => {
-      it.skip(' ', () => {
-        //
+        getHostElement().should('contain.text', labelSlot)
       })
     })
 
-    describe('(slot): hint', () => {
-      it.skip(' ', () => {
-        //
-      })
-    })
+    for (const slot of slots) {
+      describe(`(slot): ${ slot }`, () => {
+        it(`should append a '${ slot }' slot`, () => {
+          const model = ref(selectOptions[ 0 ])
+          const labelSlot = 'Hello world'
 
-    describe('(slot): counter', () => {
-      it.skip(' ', () => {
-        //
-      })
-    })
+          mountQFieldWrapper({
+            props: {
+              ...vModelAdapter(model),
+              options: selectOptions,
+              labelSlot: true
+            },
+            slots: {
+              [ slot ]: () => labelSlot
+            }
+          })
 
-    describe('(slot): loading', () => {
-      it.skip(' ', () => {
-        //
+          getHostElement().get(`.q-field__${ slot }`).should('contain.text', labelSlot)
+        })
       })
-    })
+    }
   })
 
   describe('Events', () => {
     describe('(event): clear', () => {
-      it.skip(' ', () => {
-        //
+      it('should emit the clear event when the clear button is clicked', () => {
+        const model = ref(selectOptions[ 0 ])
+        const fn = cy.stub()
+
+        mountQFieldWrapper({
+          props: {
+            ...vModelAdapter(model),
+            clearable: true,
+            options: selectOptions,
+            onClear: fn
+          }
+        })
+
+        getHostElement().get('button[type="button"]')
+          .contains(Icons.field.clear).click()
+          .then(() => {
+            expect(fn).to.be.calledWith()
+          })
       })
     })
   })
 
   describe('Methods', () => {
     describe('(method): focus', () => {
-      it.skip(' ', () => {
-        //
+      it('should focus the component', () => {
+        mountQFieldWrapper()
+
+        getHostElement()
+          .get('[tabindex="0"]')
```

Comment:
This way of selecting the input feels really strange, any reason why you went with this one?

Reply:
I have refactored to use `.get('input')` which is also as precise and better than `[tabindex="0"]`.